docs/guides/latest/advanced-use-case/pause-backup.md

```diff
@@ -0,0 +1,365 @@
+  retentionPolicy:
+    name: 'keep-last-5'
+    keepLast: 5
+    prune: true
+```
+
+Let's create the `BackupConfiguration` crd we have shown above,
+
+```console
+$ kubectl apply -f ./docs/examples/guides/latest/advanced-use-case/pause-backup/backupconfiguration.yaml 
+backupconfiguration.stash.appscode.com/pause-backup created
+```
+
+**Verify Sidecar:**
+
+If everything goes well, Stash will inject a sidecar container into the `stash-demo` Deployment to take backup of `/source/data` directory. Let’s check that the sidecar has been injected successfully,
+
+```console
+$ kubectl get pod -n demo
+NAME                          READY   STATUS    RESTARTS   AGE
+stash-demo-7489fcb7f5-jctj4   2/2     Running   0          117s
+stash-demo-7489fcb7f5-mfqps   2/2     Running   0          112s
+stash-demo-7489fcb7f5-zp8c5   2/2     Running   0          115s
+```
+
+Look at the pod. It now has 2 containers. If you view the resource definition of this pod, you will see that there is a container named `stash` which is running `run-backup` command.
+
+**Verify CronJob:**
+
+It will also create a `CronJob` with the schedule specified in `spec.schedule` field of `BackupConfiguration` crd.
+
+Verify that the `CronJob` has been created using the following command,
+
+```console
+Every 3.0s: kubectl get backupconfiguration --all-namespaces                      suaas-appscode: Thu Aug  1 17:08:08 2019
+
+NAMESPACE   NAME           TASK   SCHEDULE      PAUSED   AGE
+demo        pause-backup          */1 * * * *            27s
+```
+
+**Wait for BackupSession Succeeded:**
+
+Wait for the next schedule for backup. Run the following command to watch `BackupSession` crd,
+
+```console
+Every 3.0s: kubectl get backupssession --all-namespaces                      suaas-appscode: Thu Aug  1 17:43:57 2019
+
+NAMESPACE   NAME                      BACKUPCONFIGURATION   PHASE       AGE
+demo        pause-backup-1564659789   pause-backup          Succeeded   49s
+```
+
+We can see from the above output that the backup session has succeeded. This indicates that the volumes of the Deployment have been backed up in the backend successfully.
+
+## Pause Running Backup
+
+To stop Stash from taking backup volumes of the deployment, you can do the following things:
+
+**Patch BackupConfiguration:**
+
+- Set `spec.paused`: true in `BackupConfiguration` yaml and then apply the update. This happens:
+  - The running backup process will stop immediately and paused `BackupConfiguration` CRD will not be applicable for the targeted workload.
+  - Stash sidecar containers will not be removed from existing workloads but the sidecar will stop taking backup.
+
+You can do the things by using the following command,
+
+```console
+$ kubectl patch backupconfiguration -n demo stash-demo --type="merge" --patch='{"spec": {"paused": true}}'
+backupconfiguration.stash.appscode.com/pause-backup patched
+```
+
+Now, Describe the `BackupConfiguration` crd by using the following command,
```

Comment:
Let's describe the `BackupConfiguration` to verify that the event has been created,